mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -45,6 +45,18 @@ class mgmtsystem_action(models.Model):
     system_id = fields.Many2one('mgmtsystem.system', 'System')
     company_id = fields.Many2one('res.company', 'System', default=own_company)
 
+    stage_id = fields.Many2one(
+        'mgmtsystem.action.stage',
+        'Stage',
+        default=lambda self: self.get_default_stage()
+    )
+
+    @api.model
+    def get_default_stage(self):
+        return self.env['mgmtsystem.action.stage'].search([
+            ('is_starting', '=', True)
+        ]).id
```

Comment:
Changes in Workflow wiring are not reflected in the UI.
IMO Workflows are only useful for cross-Model processes (such as Sale -> Invoice -> Delivery).
For single-Model processes they are an overkill and can be replaced by a Kanban Board:

I would like to drop states and workflows, to replace them by Stages + Kanban Board.
History tracking and notifications are Stage will be there. Additional logic (mostly will be custom) is possible with Automated Actions.